target-chains/near/receiver/src/lib.rs

```diff
@@ -162,11 +166,12 @@ impl Pyth {
         Ok(())
     }
 
-    /// Return the deposit required to update a price feed.
-    pub fn get_update_fee_estimate(&self) -> u64 {
+    /// Return the deposit required to update a price feed. This is the upper limit for an update
+    /// call and any remaining deposit not consumed for storage will be refunded.
+    pub fn get_update_fee_estimate(&self) -> U128 {
         let byte_cost = env::storage_byte_cost();
         let data_cost = byte_cost * std::mem::size_of::<PriceFeed>() as u128;
-        4u64 * u64::try_from(data_cost).unwrap() + self.update_fee
+        (4u128 * data_cost + self.update_fee).into()
```

Comment:
Can you document why is it 4 times the data cost? (why not 5? as batches have 5 price feeds)